src/client/theme-default/components/VPNavScreen.vue

```diff
@@ -43,7 +43,7 @@ function unlockBodyScroll() {
 <style scoped>
 .VPNavScreen {
   position: fixed;
-  top: var(--vp-nav-height-mobile);
+  top: calc(var(--vp-nav-height-mobile) + 1px);
```

Comment:
We need to account for the new calculation of the navbar height, where the `--vp-nav-height-mobile` value is the height of the navbar's _excluding its border_.